src/Serialization/LieTheory.jl

```diff
@@ -1,3 +1,5 @@
+import Oscar: _vec, set_root_system_type!
```

Comment:
```suggestion
using Oscar: _vec, set_root_system_type!
```
these don't need an import as they are not extended